<!-- GAS DISTRIBUTION MAPPING with GMRF -->
<launch>
    <arg name="launch_prefix" default="" />
    
	<node pkg="gmrf_gas_mapping" exec="gmrf_gas_mapping_node" name="gmrf_node" output="screen" launch-prefix="$(arg launch_prefix)">
		<param name="frame_id" value="map" />                       ### frame where the map is published
		<param name="sensor_topic" value="/gas_data" />   			### (read) topic for gas sensor/enose		
		<param name="exec_freq" value="0.5" />                      ### (Hz) frequency to update and publish the maps		
		<param name="cell_size" value="0.1" />                      ### (m)
		
		#GMRF params
		<param name="GMRF_lambdaPrior" value="0.5" />               ### The information (Lambda) of prior factors
		<param name="GMRF_lambdaObs" value="10.0" />                ### The initial information (Lambda) of each observation (this information may decrease with time)
		<param name="GMRF_lambdaObsLoss" value="0.0" />             ### The loss of information (Lambda) of the observations with each iteration (see exec_freq)
		
		### Visualization
		<param name="colormap" value="jet" />                       ### jet, hot, cool, spring, summer, autumn, winter, bone, copper, red, green
		<param name="max_pclpoints_cell" value="20" />              ### Max number of points to plot per cell (when concentration is max)
		<param name="min_sensor_val" value="0" />                   ### For normalization
		<param name="max_sensor_val" value="1" />
	</node>

</launch>
